brainstormforce/astra
# AST-1622 - fix: Astra Meta settings not saving when custom fields option is enabled in the preferences

### Description

The Astra Meta settings were not saved due to the Custom fields option being enabled from the preference.
Here is the screenshot: https://share.bsf.io/wbu82l5e
This option you can find under the Preferences => Panels and disable it.

**How to reproduce** 

1. Go to the editor 
2. Click on the top right corner and enable the custom field from Preferences
3. Now save & reload the page.
4. Now try to change any meta setting & publish it. 
5. After that reload on the editor, a particular value is stored in a below custom field, 
6. Now again change any meta setting & publish it 
7. The result should be changed according to the meta option setting choose.

### Screenshots


- https://share.bsf.io/wbu82l5e

### Types of changes





### How has this been tested?


### Checklist:
